lib/stdlib/src/lists.erl

```diff
@@ -1546,8 +1546,8 @@ _Examples:_
 
 ```erlang
 > Fun = fun(Atom) -> atom_to_list(Atom) end.
-#Fun<erl_eval.6.10732646>
-2> lists:keymap(Fun, 2, [{name,jane,22},{name,lizzie,20},{name,lydia,15}]).
+_
```

Comment:
I wonder if seeing `_` could be confusing for those reading the docs. Perhaps we could allow it to be skipped altogether? So you would have:

```
> Fun = fun(Atom) -> atom_to_list(Atom) end.
> lists:keymap(Fun, 2, [{name,jane,22},{name,lizzie,20},{name,lydia,15}]).
```

WDYT?

Reply:
Both are allowed already, I just forgot to update the lists examples after I added that. Will fix.

Reply:
IMHO, I would keep the `_` between lines. Looks more comfortable to me not to see any suppressed line of code.

Suppressed:

```
> Fun = fun(Atom) -> atom_to_list(Atom) end.
> lists:keymap(Fun, 2, [{name,jane,22},{name,lizzie,20},{name,lydia,15}]).
[{name,"jane",22},{name,"lizzie",20},{name,"lydia",15}]
```

With the underscore:

```
> Fun = fun(Atom) -> atom_to_list(Atom) end.
_
> lists:keymap(Fun, 2, [{name,jane,22},{name,lizzie,20},{name,lydia,15}]).
[{name,"jane",22},{name,"lizzie",20},{name,"lydia",15}]
```

It looks more like the Erlang way of doing things with the underscore, but I don't know if anyone would be confused. I wonder if the suppressed line could be confusing.

Reply:
Both ways are implemented, so it is up to the author to decide which one they like the most.